docker/web/serve.json

```diff
@@ -0,0 +1,60 @@
+{
+  "directoryListing": false,
+  "trailingSlash": false,
+  "rewrites": [
+    {
+      "source": "/certification/**",
+      "destination": "/certification/index.html"
+    }
+  ],
+  "redirects": [
```

Comment:
_Unimportant Question_

Can this file not be pulled from the `nginx` repo? (I think that is where it lives...)

Reply:
This config lives here actually: https://github.com/freeCodeCamp/client-config/blob/master/serve.json - I agree it's not the best of how we have organized things, but we intend to clean up a lot of these "configs" and handle them better (make them more discoverable/maintainable).